alf/environments/suite_babyai.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) 2020 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import babyai
+import gin
+import gym
+import numpy as np
+import re
+
+import alf.environments.gym_wrappers
+from .suite_gym import wrap_env
+
+
+@gin.configurable
+def load(environment_name,
+         env_id=None,
+         max_instruction_length=32,
+         one_token_per_step=False,
+         discount=1.0,
+         max_episode_steps=None,
+         gym_env_wrappers=(),
+         alf_env_wrappers=()):
+    """Loads the selected environment and wraps it with the specified wrappers.
+
+    Note that by default a TimeLimit wrapper is used to limit episode lengths
+    to the default benchmarks defined by the registered environments.
+
+    Args:
+        environment_name (str): Name for the environment to load.
+        max_instruction_length (int): the maximum number of words of an instruction.
+        one_token_per_step (bool): If False, the whole instruction (word ID array)
+            is given in one step in the observation. If True, the word IDs are
+            given in the observation sequentially. Each step only one world ID
```

Comment:
world -> word. Should clarify that the entire instruction is given at every step.